Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7-slim
```

Comment:
I haven't kept track of what our use of various Python versions is, but know that 3.8 was released recently. Do we have a strategy for what version we use?

Reply:
Well... the app is on 3.7 hence the docker image is on 3.7. However, is there a good reason the app is not on 3.8? Absolutely not :)

Reply:
Fair enough :-)

Reply:
<cough>
https://github.com/MITLibraries/renorect/issues/4
</cough>